system/settings/settings.xml

```diff
@@ -1620,6 +1620,17 @@
           <control type="toggle" />
         </setting>
       </group>
+      <group id="3">
+      <requirement>HAS_ZEROCONF</requirement>
+        <setting id="services.deviceuuid" type="string">
```

Comment:
add `<visible>false</visible>` inside the `setting` tag to hide it in the GUI

Reply:
Thanks rechi, that worked! I was searching previously for hidden for edit, which was of course the wrong one.